src/test/java/com/wikia/webdriver/testcases/adstests/TestHorseSlots.java

```diff
@@ -0,0 +1,24 @@
+package com.wikia.webdriver.testcases.adstests;
+
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.AdsBaseObject;
+import org.testng.annotations.Test;
+
+public class TestHorseSlots extends NewTestTemplate {
+
+  @Test(groups = {"TestHorseSlots_Interstital", "TestHorseSlots"}, invocationCount = 10)
+  public void TestHorseSlots_Interstital() {
+    driver.manage().window().maximize();
+    driver.get("http://sandbox-adeng02.project43.wikia.com/wiki/SyntheticTests/Slots/InvisibleHighImpact/Interstitial");
+    AdsBaseObject ads = new AdsBaseObject(driver);
+    ads.waitForElementPresent("wikia_gpt/5441/wka.life/_project43//article/gpt/INVISIBLE_HIGH_IMPACT_2");
+  }
+
+  @Test(groups = {"TestHorseSlots_FloorAdhesion", "TestHorseSlots"}, invocationCount = 10)
+  public void TestHorseSlots_FloorAdhesion() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)